### ISSUE TITLE
Consider removing django.contrib.redirects

### ISSUE THREAD

**Issue by berkerpeksag** [python/pythondotorg#822]
Most of the redirects are in nginx: https://github.com/python/psf-salt/blob/master/salt/pydotorg/config/pydotorg.nginx.jinja Currently we have 6 redirects in the redirects app and I think we can move those to nginx. Also, it probably will reduce the number of SQL queries in return.

**Comment by malemburg** [comment]
I would agree, except that access to the salt config is far more limited than access to the Django admin interface to add such redirects.

If we can have some other way for people with Django admin access to configure such redirects, I'd be +1. This may be as easy as documenting how to submit a PR to the above configuration.
